packages/meditrak-server/src/apiV2/utilities/translateObjectFields.js

```diff
@@ -8,8 +8,8 @@ export const translateObjectFields = async (object, objectTranslators) => {
     Object.entries(object).map(async ([field, value]) => {
       if (objectTranslators[field]) {
         const newFields = await objectTranslators[field](value);
-        Object.entries(newFields).forEach(([newField, newValue]) => (object[newField] = newValue));
         delete object[field];
```

Comment:
Move the `delete` to before the line where we add the new entries to the survey response object, so that we don't go: add the translated `answers`, then delete the field `answers`